packages/components/config-provider/__tests__/config-provider.spec.ts

```diff
@@ -120,4 +120,38 @@ describe('config-provider', () => {
       ).toBeFalsy()
     })
   })
+
+  describe('button-config', () => {
+    it('limit the number of messages displayed at the same time', async () => {
+      const wrapper = mount({
+        components: {
+          [ConfigProvider.name]: ConfigProvider,
+          ElButton,
+        },
+        setup() {
+          const config = reactive({
+            max: 3,
+          })
+          const open = () => {
+            ElMessage('this is a message.')
+          }
+          return {
+            config,
+            open,
+          }
+        },
+        template: `
+          <el-config-provider :button="config">
+            <el-button @click="open">open</el-button>
+          </el-config-provider>
+        `,
+      })
+      await nextTick()
+      wrapper.find('.el-button').trigger('click')
+      wrapper.find('.el-button').trigger('click')
+      wrapper.find('.el-button').trigger('click')
+      wrapper.find('.el-button').trigger('click')
+      expect(document.querySelectorAll('.el-message').length).toBe(4)
+    })
+  })
```

Comment:
**Questions:** when you have the limitation set to max messages to `3`, why do you assert that there are 4 messages that shows up? I think this test case is completely wrong.

**Suggestion:** you should add `await` keyword before `.trigger([event])` statement.

Reply:
> **Questions:** when you have the limitation set to max messages to `3`, why do you assert that there are 4 messages that shows up? I think this test case is completely wrong.

I'm sorry, this should be some conflicts caused by another branch merged with dev. I copied it directly and forgot to modify it. 🤣 
> **Suggestion:** you should add `await` keyword before `.trigger([event])` statement.

Maybe I just need to add an `await nextTick()` after all `.trigger([event])`.